test/simulator/main/org/apache/cassandra/simulator/ClusterSimulation.java

```diff
@@ -544,7 +543,7 @@ IInstanceConfig update(IInstanceConfig config)
                    .set("concurrent_counter_writes", take(1, 4))
                    .set("concurrent_materialized_view_writes", take(1, 4))
                    .set("concurrent_reads", take(1, 4))
-                   .forceSet("available_processors", take(3, 4));
+                   .set("available_processors", take(3, 4));
```

Comment:
`forceSet` is the same as `set`... I don't remember why we added it... think this was before we refactored Yaml to support map

Reply:
We previously verified that the property existed in the yaml I think? So if a property wasn't available in earlier versions it could be an issue, or something like that.

Reply:
don't remember, it is identical logic to `set` now adays... we do not validate, we rely on `Config` and `YamlConfigLoader` to do that